### ISSUE TITLE
Post interaction buttons could use larger tap targets

### ISSUE THREAD

**Issue by fadookie** [tuskyapp/Tusky#2825]
I think the tap targets for the buttons to interact with a post (fav, boost, etc.) are too small. I find myself accidentally tapping to expand the post often when I try to use these. It's a common practice in mobile apps to increase the invisible "tap target" or "hit slop" to make buttons more usable. c.f. [Android accessibility guidelines](https://support.google.com/accessibility/android/answer/7101858?hl=en#:~:text=The%20recommended%20target%20size%20for%20touchscreen%20objects%20is%207%2D10mm.).

Please let me know if you would be open to a PR for this issue.

* * * *
- Tusky Version: 19.0
- Android Version: 13
- Android Device: Google Pixel 4a
- Mastodon instance (if applicable): N/A

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by jsoref** [comment]
Yes. I was about to file this. Oddly, I find that once I've triggered a 🌟, the hit target feels larger for that item both as 🌟 and ⭐.
